src/app/cli/src/cli_arguments.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::ops::Deref;
+
+use clap::Arg;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+pub mod add {
+    use super::CliArgument;
+
+    pub const PUBLIC: CliArgument = CliArgument::new("public");
```

Comment:
This seems a lot more typing than it's worth. I suggest to stick to the existing style for now.

I think in the near future we should instead invest the time to switch to macro-based clap configuration - I already [experimented with it](https://github.com/kamu-data/kamu-node/blob/master/src/app/oracle-provider/src/config.rs) in the oracle provider.

It's a lot more concise and type-safe.

Reply:
Ok, with a sore heart I will return string hardcoded constants :)

Reply:
Done